php/libraries/Candidate.class.inc

```diff
@@ -35,9 +35,74 @@ define('EDC_NOT_SPECIFIED', 5);
  */
 class Candidate
 {
-    var $candidateInfo;
```

Comment:
Since `$candidateInfo` is public, effectively all the candidate data before was public before and could have been changed using 

```php
Candidate->candidateInfo['field'] = $value;
```

I'm not sure if this is done in practice, but I've left these variables as being public in scope.

If this is not desirable, I can change them to be private. I'm not sure which variables (if any) should have setter functions in this case.